chef/data_bags/crowbar/bc-template-ceph.json

```diff
@@ -6,7 +6,9 @@
       "master": false,
       "disk_mode": "all",
       "config": {
-        "fsid" : ""
+        "fsid" : "",
+        "osds_in_total": 0,
```

Comment:
Ok, I removed it, but in my opinion we should left this attribute, because of some users want to configure this attribute manually.